posthog/constants.py

```diff
@@ -319,3 +319,32 @@ class FlagRequestType(StrEnum):
 PERMITTED_FORUM_DOMAINS = ["localhost", "posthog.com"]
 
 INVITE_DAYS_VALIDITY = 3  # number of days for which team invites are valid
+
+# Sync with frontend/src/scenes/surveys/constants.tsx
```

Comment:
we hould add a comment to frontend/src/scenes/surveys/constants now to sync with this one so tis both ways